packages/markdown/package.json

```diff
@@ -34,6 +34,7 @@
   "dependencies": {
     "@launchpad-ui/tokens": "workspace:~",
     "classix": "2.1.17",
```

Comment:
This will add almost [10k](https://bundlephobia.com/package/html-react-parser@3.0.8) to the bundle-size for this package, and may end up causing confusion given that we're also using `marked` ... I don't have much context for this problem, but I'm curious if we tried using the [`renderer` option from `marked`](https://marked.js.org/using_pro#renderer) as a solution?

I imagine that we'd need a way to thread that in from consumers, e.g. changes to the props surface area for the `Markdown` component, but that might allow us to avoid introducing a new parsing library and instead of a new `smartLinks` prop we'd end up with a way to more generically customize rendering behavior.

Reply:
Moved discussion to [slack](https://launchdarkly.slack.com/archives/C01TNPM36U8/p1675459228682109?thread_ts=1675459059.487959&cid=C01TNPM36U8). I'll update this thread when we come to a conclusion.